ee/ucp/admin/configure/restrict-services-to-worker-nodes.md

```diff
@@ -1,40 +1,37 @@
 ---
 title: Restrict services to worker nodes
-description: Learn how to configure Universal Control Plane to only allow running services in worker nodes.
+description: Learn how to configure Docker Universal Control Plane to only allow running services in worker nodes.
 keywords: ucp, configuration, worker
 ---
 
 >{% include enterprise_label_shortform.md %}
 
-You can configure UCP to allow users to deploy and run services only in
-worker nodes. This ensures all cluster management functionality stays
-performant, and makes the cluster more secure.
+Docker Universal Control Plane (UCP) is set to run on both manager and worker nodes by default, however, it can be configured to run only on worker nodes. This ensures that all cluster management functionality stays performant, and makes the cluster more secure.
 
-If a user deploys a malicious service that can affect the node where it
-is running, it won't be able to affect other nodes in the cluster, or
-any cluster management functionality.
+If a user deploys a malicious service that can affect the node where it is running, it cannot affect other nodes in the cluster, or any cluster management functionality.
```

Comment:
In my opinion, this line is critical enough to be drawn out as **Important**.  Also, I suggest it be edited to read as follows:

"In the event that a user deploys a malicious service capable of affecting the node on which it is running, that service will not be able to strike any other nodes in the cluster or have any impact whatsoever on cluster management functionality."